### PR TITLE
Androidx material bump

### LINKED ISSUE #12858: [Bug] [Android] Updating Xamarin.Google.Android.Material from 1.0.0.1 to 1.1.0.5 breaks style of Material components
### Description
After updating Xamarin.Google.Android.Material from 1.0.0.1 to 1.1.0.5 the Material style of components like Entry changed completely.
There are also behavioral changes, e.g. if you click on the screen, the entries somehow light up.

@PureWeen @mattleibow I was told, to tag you :-)

The Green Background color seems to be set by PlaceholderColor.

### Steps to Reproduce

1. Update Xamarin.Google.Android.Material from 1.0.0.1 to 1.1.0.5

### Expected Behavior
Style and behavior does not change after updating.

### Actual Behavior
Style and behavior does not change after updating.

### Basic Information

- Version with issue: 4.8.0.1560
- Last known good version:
- IDE:
- Platform Target Frameworks: 
  - iOS:  
  - Android:  
  - UWP:   
- Android Support Library Version: 
- Nuget Packages: Xamarin.Google.Android.Material
- Affected Devices:

### Screenshots
XamarinGoogleAndroidMaterial_Change

### Workaround

### PR DESCRIPTION
**Pull request by PureWeen**
### Description of Change ###

Update Xamarin.Google.Android.Material 1.1.0-rc3

### Issues Resolved ### 

- fixes #12858 

### Platforms Affected ### 
- Android

### Behavioral/Visual Changes ###
In theory there shouldn't be any. Compare the Visual Gallery on the previous version to this one

### Testing Procedure ###
- open visual gallery, compare to ios, and compare to previous version

### PR Checklist ###

- [ ] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

**Comment by PureWeen** [comment]
Currently blocked by
https://github.com/xamarin/AndroidX/issues/76

File: Xamarin.Forms.Material.Android/MaterialFormsTextInputLayoutBase.cs [hunk 1]
```diff
@@ -125,7 +144,7 @@ public virtual void SetHint(string hint, VisualElement element)
 				Hint = hint;
 				// EditText.Hint => Hint
 				// It is impossible to reset it but you can make it invisible.
-				EditText.SetHintTextColor(global::Android.Graphics.Color.Transparent);
+				//EditText.SetHintTextColor(global::Android.Graphics.Color.Transparent);
```
Comment (hartez) [inline Xamarin.Forms.Material.Android/MaterialFormsTextInputLayoutBase.cs:147]: Is this commenting-out intentional?

**Review by hartez — CHANGES_REQUESTED** [review]

**Comment by PureWeen** [comment]
@rmarinho I tested the nugets on this one and it pulls in the right nuget versions

```
 "Xamarin.AndroidX.Annotation/1.1.0.7": {
        "type": "package",
        "dependencies": {
          "Xamarin.AndroidX.Migration": "1.0.8"
        },
```

**Review by jsuarezruiz — CHANGES_REQUESTED** [review]
I have found some differences with the change. For example (left updated version, right 5.0.0 branch).

**Review by rmarinho — APPROVED** [review]

**Pull request merged**